packages/playback-core/src/index.ts

```diff
@@ -47,6 +69,13 @@ export const initialize = (
 ) => {
   // Automatically tear down previously initialized mux data & hls instance if it exists.
   teardown(mediaEl, core);
+  // NOTE: metadata should never be nullish/nil. Adding here for type safety due to current type defs.
+  const { metadata = {} } = props;
+  const { view_session_id = generateUUID() } = metadata;
```

Comment:
NOTE: Currently, I'm not keying the updates to these values based on `experimentalCmcd`. On the plus side, this avoids unnecessary forking paths and provides more predictable info in Mux Data for Mux Player, regardless of `CMCD` usage. On the minus side, this does mean we're "doing slightly more work" and "sending slightly more data" than is strictly required.